### PR TITLE
Improve doc for several functions

### PR DESCRIPTION
**Pull request by gpetit**
As asked in #765 we have now:

- New signature to remove existing tables for GPXRead, OSMRead and ST_Graph
- Update ST_Graph, which was quite wrong
- Update ST_Expand with new ```distance``` signature
- Update ST_RemoveRepeteadPoints with new ```tolerance``` signature

### CODE REVIEW

File: docs/dev/GPXRead.md [hunk 1]
```diff
@@ -14,13 +14,16 @@ permalink: /docs/dev/GPXRead/
...
 GPXRead(VARCHAR path, VARCHAR tableName);
+GPXRead(VARCHAR path, VARCHAR tableName, BOOLEAN deleteTables);
 {% endhighlight %}
 
 ### Description
 
 Reads a [GPX][wiki] file from `path` and creates several tables
-prefixed by `tableName` representing the file's contents.
-Which tables are produced depends on the content of the GPX file,
+prefixed by `tableName` representing the file's contents. If `deleteTables` is equal to `1`, existing tables (with the same prefix) are removed.
```
Comment (ebocher) [inline docs/dev/GPXRead.md]: is equal to true it's a boolean value

File: docs/dev/GPXRead.md [hunk 2]
```diff
@@ -48,6 +51,9 @@ CALL GPXRead('/home/user/route.gpx');
 -- * GPXDATA_TRACKSEGMENT
 CALL GPXRead('/home/user/route.gpx', 'GPXDATA');
 
+-- Existing tables starting with 'GPXDATA' will be removed
+CALL GPXRead('/home/user/route.gpx', 'GPXDATA', 1);
```
Comment (ebocher) [inline docs/dev/GPXRead.md]: nope...
CALL GPXRead('/home/user/route.gpx', 'GPXDATA', true);

File: docs/dev/OSMRead.md [hunk 3]
```diff
@@ -14,11 +14,12 @@ permalink: /docs/dev/OSMRead/
 {% highlight mysql %}
 OSMRead(VARCHAR path);
 OSMRead(VARCHAR path, VARCHAR tableName);
+OSMRead(VARCHAR path, VARCHAR tableName, BOOLEAN deleteTables);
 {% endhighlight %}
 
 ### Description
 
-Reads a [OSM][wiki] file from `path` and creates several tables prefixed by `tableName` representing the file's contents.
+Reads a [OSM][wiki] file from `path` and creates several tables prefixed by `tableName` representing the file's contents. If `deleteTables` is equal to `1`, existing tables (with the same prefix) are removed.
```
Comment (ebocher) [inline docs/dev/OSMRead.md]: deleteTables` is equal to true

File: docs/dev/OSMRead.md [hunk 4]
```diff
@@ -55,6 +56,10 @@ CALL OSMRead('/home/user/bretagne.osm.gz', 'OSM_BRETAGNE');
 -- Uses another zipped file
 CALL OSMRead('/home/user/bretagne.osm.bz2', 'OSM_BRETAGNE');
 
+-- Uses the given table name 
+-- and remove existing tables prefixed with 'OSM_BRETAGNE'
```
Comment (ebocher) [inline docs/dev/OSMRead.md]: CALL OSMRead('/home/user/bretagne.osm', 'OSM_BRETAGNE', true);

File: docs/dev/GPXRead.md (hunk 1 above)
Comment (gpetit) [inline docs/dev/GPXRead.md]: Good catch

File: docs/dev/GPXRead.md (hunk 2 above)
Comment (gpetit) [inline docs/dev/GPXRead.md]: Ok

File: docs/dev/OSMRead.md (hunk 3 above)
Comment (gpetit) [inline docs/dev/OSMRead.md]: ok

File: docs/dev/OSMRead.md (hunk 4 above)
Comment (gpetit) [inline docs/dev/OSMRead.md]: ok

**Pull request merged**